KryptoTests/CodeSigningTests.swift

```diff
@@ -0,0 +1,202 @@
+//
+//  CodeSigningTests.swift
+//  Kryptonite
+//
+//  Created by Alex Grinman on 5/19/17.
+//  Copyright © 2017 KryptCo. All rights reserved.
+//
+
+import Foundation
+
+import XCTest
+import PGPFormat
+import Sodium
+
+class CodeSigningTests: XCTestCase {
+    
+    override func setUp() {
+        super.setUp()
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+    }
+    
+    
+    func testCreatePGPPublicKeyRSA() {
```

Comment:
this is part of KeyManager which uses the `me` tag, to do this we have to separate the keymanager functions to make it testable. i'll add this to the todos